app/controllers/newsletters_controller.rb

```diff
@@ -0,0 +1,6 @@
+class NewslettersController < ApplicationController
+  def index
+    @flyers = NewsletterFinder.new.flyers
```

Comment:
I would do this a bit differently. It's hard to explain why, but I think it's preferable to do the following:

```ruby
@flyers = NewsletterFinder.new("flyers")
```

Then in `NewsletterFinder` you could `include Enumerable` which I'll let you explore on your own, but essentially if you look at your `news` and `flyers` functions they are so very similar that if you instead instantiated your object with that string then you could have a single method instead and your class would be more versatile (if you added a third type later on you wouldn't have to change your class - This is known as the Open/Closed Principle).